exercises/palindrome-products/palindrome_products_test.go

```diff
@@ -49,13 +54,17 @@ var testData = []struct {
 		Product{0, [][2]int{{-2, 0}, {-1, 0}, {0, 0}, {0, 1}, {0, 2}}},
 		Product{4, [][2]int{{-2, -2}, {2, 2}}},
 		""},
-	// or you could reverse the *digits*, keeping the minus sign in place.
+
+	or you could reverse the *digits*, keeping the minus sign in place.
 	{-2, 2,
 		Product{-4, [][2]int{{-2, 2}}},
 		Product{4, [][2]int{{-2, -2}, {2, 2}}},
-		""},
+	""},
```

Comment:
the indentation seems inconsistent here, since the last line is unindented, as oppoed to the examples immediately above (where they're the same indentation level as the `Product` lines.

If we were able to uncomment these cases (just not necessarily have them in the default set of cases to test) then we would be able to enlist the help of `go fmt` to choose the consistent style. In that light, 
https://github.com/exercism/xgo/pull/342/files#r74678867 seems like a useful route to go.